test/cql-pytest/test_secondary_index.py

```diff
@@ -345,6 +345,21 @@ def test_filter_cluster_key(cql, test_keyspace):
         rows = cql.execute(stmt)
         assert_rows(rows, [1, 1])
 
+# Reproduces #13548:
+# Create a huge index on a table and immediately recreate the table (drop and create)
```

Comment:
It's not a huge index... It's one >64K string and that's it. Please mention that it's not just a "slow view build", it's actually a specific bug for >64K string which causes the view build to never finish (you can refer to the test you copied).

Reply:
Maybe a better approach would be to actually create a huge index, and do not rely on a temp bug. A real huge index would make this test useful even after the bug is resolved. What do you think?

Reply:
If I'm inserting only strings smaller than 65535 characters, I'm unable to write such a test that would cause scylla w/o my fix to fail. That makes me believe that the fix implemented in this PR is just a workaround for the original bug (https://github.com/scylladb/scylladb/issues/8627). I think that once the original bug is solved, these changes here could be simply reverted.

Reply:
I applied your comments.